packages/babel-plugin-transform-unicode-escapes/src/index.js

```diff
@@ -0,0 +1,87 @@
+import { declare } from "@babel/helper-plugin-utils";
+import { types as t } from "@babel/core";
+
+export default declare(api => {
+  api.assertVersion(7);
+
+  const surrogate = /[\ud800-\udfff]/g;
+  const unicodeEscape = /(\\+)u\{([0-9a-fA-F]+)\}/g;
+
+  function escape(code) {
+    let str = code.toString(16);
+    // Sigh, node 6 doesn't have padStart
```

Comment:
Maybe add a "TODO" in the comment so that we don't forget about it for Babel 8.

Reply:
Done.